data/human/human missions.txt

```diff
@@ -7455,3 +7455,573 @@ mission "Adelita 5"
+				`	"Thanks for the overview, but I'm still not interested. I'm out."`
+					decline
+			label accepting
+			`	Howard nods with approval. "Excellent. The Marauder ship was last seen near Oblate, in the outer rim of that system. It's likely to be well armed, so stay sharp. Once you've disabled it, head back here and I'll make sure you're properly compensated."`
+			branch blewupgolden
+				has "syndicate bus golden jewel dead"
+			`	He cracks a smile. "I remember the first time we met you left that Vanguard alive. Same deal here, but disable the Marauder. Hopefully that Vanguard's not still around.`
+				goto ending
+			label blewupgolden
+			`	He leans in slightly, lowering his voice. "Just be careful. We don't want to destroy the ship. Cripple it so we can study it, but don't blow it into oblivion like you did to that Vanguard when we first met.`
+			label ending
+			`	"Well, that's all I have for you. Goodbye and good luck!"`
+				accept
+
+	on enter "Moktar"
+		"reputation: Independent (Killable)" = -1000
+	npc
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"
+		to despawn
+			has "syndicate bus golden jewel dead"
+		on encounter
+			conversation
+				`When you enter <waypoints>, your sensors detect the same pirate Vanguard from when you first rescued Howard. Your ship is immediately hailed, and the pirate opens communication.`
+				`	"I should have destroyed you when I had the chance. Prepare to meet your end!"`
+				choice
+					`	"Maybe I should have destroyed you!"`
+					`	"Uhhh... who are you again?"`
+						goto whomst
+				`	"Argh, you slithering, spineless snake! I'll tear your ship apart piece by piece!"`
+					goto power
+				label whomst
+				`	"Bah, you banal, bumbling baboon! Prepare to meet your doom!"`
+				label power
+				`	You can't help feeling a sense of deja vu as the Vanguard's engines roar to life as you see over half a dozen fixed guns take aim at your ship, ready to strike.`
+		on kill
+			set "syndicate bus golden jewel dead"
+			conversation
+				`The crew of the Golden Jewel are no more. You think you can make out the pirate captain drifting in space nearby.`
+				choice
+					`	Best not to dwell on these things.`
+
+	on enter "Oblate"
+		"reputation: Independent (Killable)" = -1000
+	npc disable save
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Oblate"
+		ship "Marauder Quicksilver (Engines) (Safe)" "Pilgrim Shadow"
+		on disable
+			conversation
+				`The ship has been disabled. You receive a message from Howard almost immediately:`
+				`	"<first> <last>, you've done it again! Meet back at <destination> to discuss."`
+				`		Howard Vendeur`
+				`		Investor Relations Manager, Megaparsec Incorporated`
+		on kill
+			conversation
+				`The ship has been destroyed. Like before, you almost immediately receive a message from Howard:`
```

Comment:
Like before?

Reply:
The ship has to be disabled before it's destroyed, so in game if the player overshoots and destroys it, they would get the `on disable` message and then a few seconds after the `on kill` one